common/config/presets/collectibles.toml

```diff
@@ -0,0 +1,6 @@
+[collectibles]
+########################################################################################################################
+#                                                                                                                      #
+#                                          Collectibles Configuration Options                                          #
+#                                                                                                                      #
+########################################################################################################################
```

Comment:
An empty line is needed at the end of the file please, for when the individual files are stringed together to make the whole.